### ISSUE TITLE
Proposal: Use accessibilityjs in development to highlight issues

### ISSUE THREAD

**Issue by ThisIsMissEm** [tootsuite/mastodon#7234]
Someone showed me this the other day, and I think it'd be worth a shot for trying to improve the accessibility of the Web UI.

https://github.com/github/accessibilityjs

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Gargron** [comment]
We already have react-a11y, it's part of JS code linting and it highlights issues like images without alt attribute or elements that should be buttons but aren't. There's an .eslintrc in the repository so if you have the right sublime text plugin installed (i think its sublimelinter) you get these warnings live as you type

**Comment by ThisIsMissEm** [comment]
Oh! I had no idea about those — I use VS Code as my editor, and it didn't seem to pick them up.

**Comment by mal0ki** [comment]
@ThisIsMissEm Unless you want help solving the issue of it not picking it up, I'll be closing this issue, as we are using an equivalent.

**Comment by ThisIsMissEm** [comment]
That's fine — I've been away from my computer for a little while, and meant to close this anyway.
